frontend/pages/DashboardPage/cards/HostsEnrolledCard/_styles.scss

```
@@ -45,18 +45,15 @@
   }
 
   &__tooltip {
-    background: $core-fleet-black;
-    color: $core-fleet-white;
-    padding: $pad-small $pad-medium;
-    border-radius: $border-radius;
-    font-size: $xx-small;
+    @include tooltip-text;
     box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.1);
+    // Bar hover shows short "N hosts" / "X% of fleet" values that fit on one
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)